docs/sources/reference/commandline/cli.md

```diff
@@ -533,6 +533,10 @@ considered as normal files. Filepaths in .dockerignore are absolute with
 the current directory as the root. Wildcards are allowed but the search
 is not recursive.
 
+When the `URL` references a Dockerfile, or a Dockerfile is read from 
+`STDIN`, then the `-f` option will have no effect and the contents of the
+Dockerfile will be placed in a file called `Dockerfile`.
```

Comment:
Might help to qualify which file you're talking about and break the material up into separate sentences for separate thoughts. E.g.: 
"When a Dockerfile is referenced by `URL` or read from `STDIN`, the `-f` option will have no effect. The contents of the referenced Dockerfile will be copied into a new, local file, also called `dockerfile`.